jsondb.js

```diff
@@ -81,8 +81,30 @@ export const tryPost = ({ ...props }) => {
   if (props.method !== "POST") {
     return props;
   }
-  const { lensPath, json } = props;
-  return { data: "post" };
+  const { lensPath, json, body } = props;
+  const parentPath = [...lensPath];
+  const parentView = U.view(U.lensPath(parentPath))(json);
+  if (parentPath.length === 0) {
```

Comment:
We can use `isEmpty` util fixed in PR #9 to remove length comparison and after changes are merged I can name it to `isRootPath` for example (because it checks whether the user hit `/api` route).

Reply:
isRootPath sounds good